addons/mail/static/src/components/follower_list_menu/follower_list_menu.xml

```diff
@@ -6,7 +6,9 @@
             <div class="o_FollowerListMenu_followers" t-ref="dropdown">
                 <button class="o_FollowerListMenu_buttonFollowers btn btn-link" t-att-disabled="props.isDisabled" t-on-click="_onClickFollowersButton" title="Show Followers">
                     <i class="fa fa-user"/>
-                    <span class="o_FollowerListMenu_buttonFollowersCount pl-1" t-esc="thread.followers.length"/>
+                    <t t-if="thread and thread.followers">
```

Comment:
template should have a top level guard on thread, not here (and not followers which is always set if thread is set)